docs/ide-usage.md

```diff
@@ -152,19 +131,16 @@ If you do not wish to use the automated setup, you can follow the [manual setup]
    1. Check `Use auto-import` and uncheck `Create separate module per source set`.
    1. Ensure `Create directories for empty content root automatically` is unchecked.
    1. Ensure `Use default gradle wrapper` is selected.
-   1. Ensure for `Gradle JVM:` that a JDK 7 with a name of `1.7` is selected.
+   1. Ensure for `Gradle JVM:` that a JDK 8 with a name of `1.8` is selected.
    1. Click `Finish`.
    1. Create a `build` folder in your project root if it does not exist while waiting for IntelliJ to finish indexing.
    1. If you used a different project root directory other than `teammates`, you will receive a dialog box with the
```

Comment:
Does this still happen now that I have removed `settings.gradle` which imposed this rule?

Reply:
Oops didn't check this 😅